src/components/SmartMeterReadingsChart.scss

```diff
@@ -0,0 +1,28 @@
+.smartMeterReadingsChart {
+    & .button-switcher {
+        box-shadow: 0 2px 10px rgba(0, 0, 0, 0.267125);
+    
+        button {
```

Comment:
Please avoid using such general declarations for SCSS. It is bad practice and results in bad performance long term.

https://csswizardry.com/2011/09/writing-efficient-css-selectors/